frontend/src/bundles/common/components/header/header.tsx

```diff
@@ -14,6 +16,12 @@ type HeaderProperties = {
 
 const Header = ({ toggleSidebar }: HeaderProperties): JSX.Element => {
     const { theme } = useAppSelector((state) => state.theme);
+    const { user } = useAppSelector(({ auth }) => ({
+        user: auth.user as UserAuthResponseDto,
+    }));
```

Comment:
```suggestion
    const { user } = useAppSelector(({ auth }) => auth);
```

Reply:
Make like this to avoid warnings from redux

Reply:
Fixed. Thanks!